src/RestApi/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -175,7 +189,7 @@ protected function get_properties() {
 					$this->get_store_currency_properties(),
 					[
 						'line_subtotal'     => [
-							'description' => __( 'Line price subtotal (excluding coupons and discounts).', 'woo-gutenberg-products-block' ),
```

Comment:
Why was this removed? It's still accurate afaik.

Reply:
True! I've tweaked it (7d50a6f5ebc864cc86a77b1d09502c5026e1423a), as this excludes any coupons that apply to the cart, but does include sale prices. Also clarified the description for `line_total` in 143230765838c525fa19367255926c3fbf6f47c6.